disco_aws_automation/disco_deploy.py

```diff
@@ -51,9 +52,15 @@ def _get_hostclasses_from_pipeline_definition(self, pipeline_definition):
         hostclasses = {entry["hostclass"]: entry for entry in pipeline_definition}
 
         for hostclass, entry in hostclasses.iteritems():
-            for definition in ["max_size", "desired_size", "min_size"]:
-                if definition in entry:
-                    hostclasses[hostclass][definition] = int(entry[definition])
+            if "min_size" in entry:
+                hostclasses[hostclass]["min_size"] =\
+                    int(DiscoAWS._size_as_minimum_int_or_none(entry["min_size"]))
+            if "desired_size" in entry:
+                hostclasses[hostclass]["desired_size"] =\
+                    int(DiscoAWS._size_as_maximum_int_or_none(entry["desired_size"]))
+            if "max_size" in entry:
+                hostclasses[hostclass]["max_size"] =\
```

Comment:
i see make sense, but i still think that something like this:
```python
for hostclass in hostclasses.iteritems():
    if ... :
        hostclass["key"] = value
```
is simpler and better to read code.